android/app/src/main/java/org/fossasia/openevent/viewmodels/SearchActivityViewModel.java

```diff
@@ -0,0 +1,111 @@
+package org.fossasia.openevent.viewmodels;
+
+import android.arch.lifecycle.ViewModel;
+import android.arch.lifecycle.LiveData;
+import android.arch.lifecycle.MutableLiveData;
+
+import org.fossasia.openevent.data.Microlocation;
+import org.fossasia.openevent.data.Session;
+import org.fossasia.openevent.data.Speaker;
+import org.fossasia.openevent.data.Track;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import io.realm.Case;
+import io.realm.Realm;
+import io.realm.RealmResults;
+import timber.log.Timber;
+
+public class SearchActivityViewModel extends ViewModel {
+
+    private String searchText = "";
+    private List<Object> objectList = new ArrayList<>();
+    private Realm realm;
+
+    public SearchActivityViewModel() {
+        realm = Realm.getDefaultInstance();
+    }
+
+    public LiveData<List<Object>> getSearchResults() {
+        objectList.clear();
+        MutableLiveData<List<Object>> searchResults = new MutableLiveData<>();
```

Comment:
You're creating a LiveData everytime the method is called. Make it global and do it only once.